php/class-support.php

```diff
@@ -0,0 +1,226 @@
+<?php
+/**
+ * Cloudinary Logger, to collect logs and debug data.
+ *
+ * @package Cloudinary
+ */
+
+namespace Cloudinary;
+
+use Cloudinary\Component\Setup;
+use Cloudinary\Settings\Setting;
+
+/**
+ * Plugin logger class.
+ */
+class Support extends Settings_Component implements Setup {
+
+	/**
+	 * Holds the plugin instance.
+	 *
+	 * @var     Plugin Instance of the global plugin.
+	 */
+	public $plugin;
+
+	/**
+	 * Holds the option key for tracking reports.
+	 */
+	const REPORT_KEY = '_cloudinary_report';
+
+	/**
+	 * Logger constructor.
+	 *
+	 * @param Plugin $plugin Global instance of the main plugin.
+	 */
+	public function __construct( Plugin $plugin ) {
+		parent::__construct( $plugin );
+		add_action( 'cloudinary_settings_save_setting_enable_support', array( $this, 'init_reporting' ), 10, 3 );
+		add_filter( 'media_row_actions', array( $this, 'add_inline_action' ), 10, 2 );
+		add_filter( 'post_row_actions', array( $this, 'add_inline_action' ), 10, 2 );
```

Comment:
@DavidCramer 

The option `Add to Cloudinary Report` is not always the last. What do you think if we bump the priority here to ensure that it's always the last?